src/component/utils/getElementForBlockType.js

```diff
@@ -12,32 +12,20 @@
 
 'use strict';
 
+const DefaultDraftBlockRenderMap = require('DefaultDraftBlockRenderMap');
+
 import type {DraftBlockType} from 'DraftBlockType';
+import type {DraftBlockRenderMap} from 'DraftBlockRenderMap';
+
+function getElementForBlockType(
+  blockType: DraftBlockType,
+  customBlockMap: DraftBlockRenderMap
+): string {
+  const draftBlockRenderMap = customBlockMap !== undefined ? customBlockMap : DefaultDraftBlockRenderMap;
 
-function getElementForBlockType(blockType: DraftBlockType): string {
-  switch (blockType) {
-    case 'header-one':
-      return 'h1';
-    case 'header-two':
-      return 'h2';
-    case 'header-three':
-      return 'h3';
-    case 'header-four':
-      return 'h4';
-    case 'header-five':
-      return 'h5';
-    case 'header-six':
-      return 'h6';
-    case 'unordered-list-item':
-    case 'ordered-list-item':
-      return 'li';
-    case 'blockquote':
-      return 'blockquote';
-    case 'media':
-      return 'figure';
-    default:
-      return 'div';
-  }
+  return draftBlockRenderMap[blockType] && draftBlockRenderMap[blockType].element ?
+    draftBlockRenderMap[blockType].element :
+    draftBlockRenderMap.unstyled.element;
```

Comment:
What if `unstyled` does not have a defined element? Thinking this should probably fall back to `div`.